src/wp-includes/user.php

```diff
@@ -4951,3 +4951,44 @@ function wp_is_application_passwords_available_for_user( $user ) {
 	 */
 	return apply_filters( 'wp_is_application_passwords_available_for_user', true, $user );
 }
+
+/**
+ * Register the user meta for persisted preferences.
```

Comment:
Can we give a little context to what "persisted preferences" means? e.g. mention that the block editor uses it.

Reply:
I've updated the comment, let me know if you think that looks ok.

Reply:
Much better LGTM 👍